package.json

```
@@ -407,6 +407,7 @@
     "@types/gulp-autoprefixer": "^0.0.33",
     "@types/gulp-dart-sass": "^1.0.1",
     "@types/gulp-sourcemaps": "^0.0.35",
+    "@types/jest": "29.1.2",
```

Could we use a range here?
```suggestion
    "@types/jest": "^29.1.2",
```